### DISCUSSION TITLE
lost audio

### DISCUSSION THREAD

**Discussion by unevoljitelj** [Joshua-Riek/ubuntu-rockchip#114] [General]
after few days not using opi5, turned it on and no sound.
under settings / sound there is nothing under output or input devices. 
not sure what could cause it. looking for help on how to start fixing it.

**Comment by Joshua-Riek** [comment]
A package update caused some conflict with a config file I set.

 Try the below commands then reboot your system:
```
git clone https://github.com/Joshua-Riek/ubuntu-orange-pi5.git
sudo cp -r ubuntu-orange-pi5/overlay/usr/share/alsa /usr/share
sudo cp -r ubuntu-orange-pi5/overlay/usr/share/pulseaudio /usr/share
```

**Reply by unevoljitelj** [reply]
fixed. thumbs up! i apologize if i ask a dumb question on occasion, still getting used to linux, altho ive been using it somewhat for a year or two.

**Reply by Joshua-Riek** [reply]
Don't worry, I'm more than happy to help! :smile:

**Reply by unevoljitelj** [reply]
is it ok to get you a coffee via sponsor button? :) or some other way

**Reply by Joshua-Riek** [reply]
Haha sure! :)

**Reply by JoaoLMM** [reply]
Hey i am experiencing the same issue but /alsa and /pulseaudio no longer exist. is there a way to make audio work again?

**Reply by Joshua-Riek** [reply]
Do you know what version you have installed? I did an audio re-work in the v1.14 release.

**Reply by Joshua-Riek** [reply]
I assume an older version, you can try the below commands:
```bash
git clone -b v1.13 https://github.com/Joshua-Riek/ubuntu-orange-pi5.git
sudo cp -r ubuntu-orange-pi5/overlay/usr/share/alsa /usr/share
sudo cp -r ubuntu-orange-pi5/overlay/usr/share/pulseaudio /usr/share
```

**Reply by JoaoLMM** [reply]
I have version 1.12 installed. What you proposed worked (i changed version to 1.12). Thank you for the quick reply!
